assets/i18n/es.po

```diff
@@ -0,0 +1,720 @@
+msgid "Search for \"%1\""
+msgstr "Buscar por "%1"" \"%1\""
+
+#: assets/components/ConnectionForm.svelte:141
+msgid "Secure connection (TLS)"
+msgstr "Conexión segura (TLS)"
+
+#: assets/page/SettingsAdminUsers.svelte:77
+msgid "See all users"
+msgstr "Ver todos los usuarios"
+
+#: assets/page/ConversationAdd.svelte:62
+msgid "Select..."
+msgstr "Seleccionar..."
+
+#: assets/page/Login.svelte:82
+msgid "Set new password"
+msgstr "Establecer nueva contraseña"
+
+#: assets/components/ChatSidebar.svelte:202
+msgid "Settings"
+msgstr "Ajustes"
+
+#: assets/page/Help.svelte:27
+msgid "Shift focus between chat input and search in sidebar."
+msgstr "Cambiar el foco entre la entrada del chat y la búsqueda en la barra lateral."
+
+#: assets/page/Help.svelte:24
+msgid "Shortcuts"
+msgstr "Atajos"
+
+#: assets/components/ConnectionForm.svelte:149
+msgid "Show advanced settings"
+msgstr "Mostrar ajustes avanzados"
+
+#: assets/page/ConversationAdd.svelte:90
+msgid "Showing %1 of %2 conversations, but the list is still loading."
+msgstr "Mostrando %1 de %2 conversaciones, pero la lista aún está cargando."
+
+#: assets/page/ConversationAdd.svelte:88
+msgid "Showing %1 of %2 conversations."
+msgstr "Mostrando %1 de %2 conversaciones."
+
+#: assets/page/Login.svelte:111
+msgid "Sign in"
+msgstr "Iniciar sesión"
+
+#: assets/page/Login.svelte:88
+msgid "Sign up"
+msgstr "Registrar"
+
+#: assets/page/Help.svelte:70
+msgid "Source code"
+msgstr "Código fuente"
+
+#: assets/page/Chat.svelte:159
+msgid "Start video conference"
+msgstr "Iniciar video conferencia"
+
+#: assets/page/Fallback.svelte:29
+msgid "Started Convos."
+msgstr "Convos ha iniciado."
+
+#: assets/page/Chat.svelte:170
+msgid "Started chatting on %1"
+msgstr "Se inició conversación en %1"
+
+#: assets/page/Help.svelte:41
+msgid "Text formatting"
+msgstr "Formato de texto"
+
+#: assets/page/Fallback.svelte:37
+msgid "The Convos Team have been searching and searching, but the requested page could not be found."
+msgstr "El Equipo de Convos ha estado buscando y buscando, pero la página solicitada no se pudo encontrar."
+
+#: assets/page/Help.svelte:33
+msgid "The following rules apply when typing a message:"
+msgstr "Las siguientes reglas aplican al escribir un mensaje:"
+
+#: assets/page/Login.svelte:51
+msgid "The link has expired."
+msgstr "El enlance ha expirado."
+
+#: assets/page/Login.svelte:51
+msgid "The link is invalid."
+msgstr "El enlace es inválido."
+
+#: assets/page/SettingsAdmin.svelte:95
+msgid "The path part is the default channel to join. \"%%23convos\" means \"#convos\"."
+msgstr "La parte de la ruta es el canal a unirse por defecto. \"%%23convos\" significa \"#convos\"."
+
+#: assets/page/SettingsAccount.svelte:92
+msgid "Theme"
+msgstr "Tema"
+
+#: assets/page/SettingsAdmin.svelte:88
+msgid "This email can be used by users to get in touch with the Convos admin."
+msgstr "Este correo electrónico puede ser utilizado ppara contactarse con el administrador de Convos."
```

Comment:
Typo: `s/ppara/para`

Maybe "puede usarse"? That sounds more natural to me, but I'm not from Spain (nor am I sure what variety of Spanish is the target). The current version is fine too.

Reply:
Maybe "se puede utilizar" sounds better. Less formal and more direct.

Reply:
What about "puedes utilizar este correo para contactarte con el administrador de Convos"?

Reply:
I like that last one too :+1:

Reply:
No, wait, I got the context. It doesn't fit. It's an admin setting, not a message for users.

Reply:
I think this is better: "Este correo electrónico se puede utilizar para contactars al administrador de Convos."